config.yaml.dist

```diff
@@ -113,14 +113,21 @@ google_maps_api:
     url: https://maps.googleapis.com/maps/api/distancematrix/json?origins={origin}&destinations={dest}&mode={mode}&sensor=true&key={key}&arrival_time={arrival}
     enable: False
 
-# Register at 2captcha and enter your API key below. you will also
-# have to install a Chrome Web Driver and write below the path to
-# the executable.
-captcha:
-  api_key: YOUR_API_KEY
-  driver_path: YOUR_CHROME_DRIVER_PATH
-  driver_arguments:
-    - "--headless"
+# If you are planning to scrape immoscout24.de, the bot will need 
+# to circumvent the sites captcha protection by using a captcha 
+# solving service. Register at either imagetypers or 2captcha 
+# (the former is prefered), desposit some funds, uncomment the 
+# corresponding lines below and replace your API key/token.
+# you will also have to install a Chrome Web Driver and write below 
+# the executable path, the driver_arguments can be left as is.
+# captcha:
+#       imagetypers:
+#             token: alskdjaskldjfklj
+#       2captcha:
+#             api_key: alskdjaskldjfklj
```

Comment:
Should we maybe give a better name to the placeholder?

Reply:
Good point. Pull requests welcome :) And I agree with your point about backwards-compatibility. It's not really something I've had on my radar, but it makes sense to take more care of that in the future. Sorry for not spotting it.